src/main/java/util/Brackets.java

```diff
@@ -0,0 +1,31 @@
+package util;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Brackets {
+    public static final Map<String, String> openBrackets;
+    public static final Map<String, String> closeBrackets;
+
+    static {
+        openBrackets = new HashMap<>();
+        closeBrackets = new HashMap<>();
+        openBrackets.put("(", ")");
+        closeBrackets.put(")", "(");
+    }
```

Comment:
```suggestion
    public final static Map<String, String> openBrackets= new HashMap<>() {{put("(", ")");}};
    public final static Map<String, String> closeBrackets= new HashMap<>() {{put(")", "(");}};
```

처럼 사용하면 생성과 동시에 초기화를 할 수 있어요~

Reply:
오👀 제가 잘 모르고 있었네요 감사합니다!!